src/app/components/activity/feed/devlog-post/text/text.ts

```diff
@@ -76,9 +79,5 @@ export default class AppActivityFeedDevlogPostText extends Vue {
 		}
 
 		this.feed.setItemOpen(this.item, false);
-
-		setTimeout(() => {
-			this.isToggling = false;
-		}, 1000);
```

Comment:
Was that meant purely to give the animation time to complete or something? why was that even there?

Reply:
This way old code that I noticed we didn't need anymore since it had no effect. It used to animate the expanding and collapsing, but now it's instant.